examples/assets/config/server/sparql_workspace_server.json

```diff
@@ -0,0 +1,5 @@
+{
+  "url": "http://localhost:9013/blazegraph/",
```

Comment:
same create an issue to get remind to change this file

Reply:
documented in #728